cypress/integration_defunct/logistics/materialentnahme_complete_HUs.js

```diff
@@ -0,0 +1,178 @@
+import { BPartner } from '../../support/utils/bpartner';
```

Comment:
What's the reason we're moving this to `defunct` ? Date widget issues or something else ?

Reply:
This test has been moved back in the working suite but in another branch. It was initially created in integration/defunct as I had to clarify some aspects with @metas-dh